haas/test_common.py

```diff
@@ -286,6 +287,33 @@ def undefine_headnodes():
 
     request.addfinalizer(undefine_headnodes)
 
+def additional_db():
```

Comment:
iirc, this is just adding more database objects needed by the new stuff that weren't already in `initial_db`, correct? I'd explain the situation, perhaps starting with a one liner saying it populates the databse with initial_db + objects for testing network acls, and then a paragraph or two explaining why it needs to be an add on function, rather than just being added to initial_db (migration tests). Might make sense to add some working to the docstring for `initial_db` mentioning that the set of objects it creates can't be changed and why.